src/tendencies/implicit/implicit_tendency.jl

```diff
@@ -60,91 +69,160 @@ function vertical_transport!(ᶜρcₜ, ᶠu³, ᶜρ, ᶜρc, p, ::Val{:zalesak
         )
 end
 
-#=
-
-TODO: can we delete this?
-
-# Used for automatically computing the Jacobian ∂Yₜ/∂Y. Currently requires
-# allocation because the cache is stored separately from Y, which means that
-# similar(Y, <:Dual) doesn't allocate an appropriate cache for computing Yₜ.
+implicit_vertical_advection_tendency!(Yₜ, Y, p, t, colidx) =
```

Comment:
Should we still be coding it in a way that allows us to run the EDMFX model fully explicit (for debugging?)